js/renovation/viz/core/tooltip.tsx

```diff
@@ -201,7 +205,17 @@ export class TooltipProps {
 
   @OneWay() interactive = false;
 
+  @OneWay() enabled = true;
+
+  @OneWay() shared = false;
+
+  @OneWay() location = 'center';
+
+  @OneWay() zIndex?: number;
+
```

Comment:
Why you did not specify a property type?

```ts
  @OneWay() enabled: boolean = true;

  @OneWay() shared: boolean = false;

  @OneWay() location: locationType = 'center';

  @OneWay() zIndex?: number;
```

Reply:
The `shared` and `enabled` options have default values and it is clear what the type. The `location` option, I agree, should have the type